build_library/build_image_util.sh

```diff
@@ -686,7 +687,23 @@ EOF
         "${BUILD_DIR}/${image_kconfig}"
   fi
 
+  # Build the selinux policy
+  if pkg_use_enabled coreos-base/coreos selinux; then
+      sudo chroot "${root_fs_dir}" bash -c "cd /usr/share/selinux/mcs && semodule -s mcs -i *.pp"
+  fi
+
   write_contents "${root_fs_dir}" "${BUILD_DIR}/${image_contents}"
```

Comment:
@krnowak I didn't move the `write_contents` here with the idea that for the image contents /etc is the path that makes more sense. I'm totally fine moving this below to have the real image contents with `/usr/share/…etc`.